allowed-breaking-changes.txt

```diff
@@ -61,3 +61,26 @@ weakened:@aws-cdk/cloud-assembly-schema.FileSource
 # These are fine, since they shouldn't be widely used.
 weakened:@aws-cdk/core.FileAssetLocation
 weakened:@aws-cdk/aws-events.RuleTargetConfig
+
+# Changes needed for moving the Action class from the codepipeline-actions
+# to the codepipeline module
+base-types:@aws-cdk/aws-codepipeline-actions.AlexaSkillDeployAction
```

Comment:
Fairly sure this is not okay for Java users.

Can you not leave `export class Action extends codepipeline.Action { }` in the `codepipeline-actions` module?

Reply:
We need to `@deprecated` it, and then probably also add it the strip-deprecated exclude list (if that already exists).

Reply:
Ok. Left the "old" `Action` from the `codepipeline-actions` module in place.

There's no exclude list for deprecations, so made it 'stable' instead.